examples/manageddisk_lustre/config.json

```diff
@@ -0,0 +1,257 @@
+{
+  "location": "westeurope",
```

Comment:
this is a hard coded value when there is a variable for it defined below

Reply:
Changed to "<NOT-SET>", but this should be done with lustre_full too then.

Reply:
lustre_full has a hardcoded location at the top but it doesn't have an unused location variable.  Having an variable would confuse someone as they would expect it to actually change the location.

Reply:
changed